docs/.vuepress/public/download-preselect.js

```diff
@@ -0,0 +1,19 @@
+(function() {
+    // Only run if we are on the downloads page
+    const url = window.location.pathname;
+    if (url.indexOf('download.html') === -1) { return }
+    // Supported targets that correspond to the table headers (case-sensitive)
+    const targets = [
+        {agent: 'Windows', header: 'Windows'}, 
+        {agent: 'Mac OS X', header: 'macOS'},
+        {agent: 'Linux', header: 'Linux'}
+    ];
+    // Obtain the browser's user agent
+    const agent = navigator.userAgent;
+    // Get all details accordion tables
+    const detailsElements = Array.from(document.querySelectorAll('details.custom-container.details'));
+    // Find an instance of the target in the user agent (case-insensitive)
+    const target = targets.find(i => agent.toLowerCase().indexOf(i.agent.toLowerCase()) !== -1);
+    // Set the open property on the element who's summary header corresponds to the found target
+    detailsElements.find(i => i.childNodes[0].innerHTML === target.header)?.setAttribute('open', '');
```

Comment:
```suggestion
    detailsElements.find(i => i.childNodes[0].innerHTML === target?.header)?.setAttribute('open', '');
```

So I've tested this rather quickly [here](https://jsfiddle.net/ktsuv2e8/11/#&togetherjs=ScFLxPol2z) and wanted to recommend we use optional chaining when referencing the `header` object, just in case we get a UserAgent we don't expect, so that we don't have an error, and instead just get `undefined`.